apps/admin/frontend/src/components/reporting/filter_editor.tsx

```diff
@@ -47,17 +47,15 @@ const RemoveButton = styled(Button)`
   width: 1rem;
 `;
 
-const FILTER_TYPES = [
-  'precinct',
-  'voting-method',
-  'ballot-style',
-  'scanner',
-  'batch',
-  'party',
-  'adjudication-status',
-  'district',
-] as const;
-export type FilterType = (typeof FILTER_TYPES)[number];
+export type FilterType =
```

Comment:
This and ones like it is because `@typescript-eslint/no-unused-vars` now considers values to be unused if they are only used for their types.

Reply:
the style gods are so fickle